packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,689 @@
+| `lastUpdated` | String (ISO-8601)                                    | Timestamp when the factor profile feature was last modified. |
+| `_links`      | [Links Object](#factor-profile-feature-links-object) | Link relations for this object                               |
+
+#### Adoption Factor Profile Feature Object
+
+The adoption feature is available for all factor profiles. It controls the settings that determine when a factor may be configured, or "adopted", for a user.
+
+##### Example
+
+```json
+{
+        "type": "adoption",
+        "id": "fpfuicb8700wW0Jq20g3",
+        "created": "2019-10-08T18:19:04.000Z",
+        "lastUpdated": "2019-10-10T16:11:43.000Z",
+        "cardinality": {
+            "min": 0,
+            "max": 1
+        },
+        "selfService": {
+            "eligibility": "ALLOWED",
+            "verificationMethod": {
+                "type": "ANY_FACTOR"
+            }
+        },
+        "_links": {
+            "self": {
+                "href": "http://{yourOktaDomain}/api/v1/org/factors/{factorType}/profiles/fpruib7klOvW4pAuK0g3/features/fpfuicb8700wW0Jq20g3",
+                "hints": {
+                    "allow": [
+                        "GET",
+                        "PUT",
+                        "DELETE"
+                    ]
+                }
+            }
+        }
+    }
+```
+
+##### Properties
+
+| Property | Type   | Description                                                          |
+| -------- | ------ | -------------------------------------------------------------------- |
+| `cardinality.min`    | Number | The minimum number of factor instances user must enroll in.          |
+| `cardinality.max`    | Number | The maximum number of factor instances user is allowed to enroll in. |
+| `selfService.eligiblity`      | String  | Indicates if factor may be adopted. Possible values: `ALLOWED`, `NOT_ALLOWED` |
+| `selfService.verificationMethod.type`      | String | Indicates which factors may be used to verify an adoption operation. Possible values: `ANY_FACTOR`, `CHAIN` |
+
+
+###### Chain verification
+Chain verification may be used to specify which factor may be used to verify the recovery
+
+###### Example
+```json
+{
+        "type": "adoption",
+        "id": "fpfuicb8700wW0Jq20g3",
+        "created": "2019-10-08T18:19:04.000Z",
+        "lastUpdated": "2019-10-10T16:11:43.000Z",
+        "cardinality": {
+            "min": 0,
+            "max": 1
+        },
+        "selfService": {
+            "eligibility": "ALLOWED",
+            "verificationMethod": {
+                "type": "CHAIN",
+                "chains": [
+                    {
+                        "criteria": [
+                            {
+                                "factorProfileId": "fpruihGXyLzyGYCq30g3"
+                            }
+                        ]
+                    }
+                ],
+                "noAuthNEmptyChain": false
```

Comment:
This shouldn't be visible, I believe this was `@JsonIgnored` in the past week

Reply:
Same in the other examples

Reply:
+1 Yep, this got removed so should not be part of the examples